.circleci/config.yml

```diff
@@ -39,40 +43,39 @@ jobs:
       - run:
           name: Install deps
           command: |
-            npm install
+            yarn install
             sudo apt-get install poppler-utils
       - run:
           name: List deps
           command: |
-            npm ls
+            yarn list
             pdftops -v
       - save_cache:
           paths:
             - node_modules
           key: v{{ .Environment.CIRCLE_CACHE_VERSION }}-deps-{{ .Branch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "package.json" }}
```

Comment:
If I'm not mistaken using `{{ .Environment.CIRCLE_JOB }}` defeats the purpose of having a cache.
Falcon  uses `key: v2-yarn-{{ checksum "yarn.lock" }}`.
`key: v2-yarn-{{ checksum "package-lock.json" }}` should work with `npm`.

Reply:
@n-riesco @etpinard Oh my, I am embarrassed at such a silly mistake!

Okay, I have updated `config.yml` to use the recommended key for Yarn (if/when we revert to NPM, I'll use the NPM key instead).

What do you think?